code/_onclick/item_attack.dm

```diff
@@ -66,12 +66,6 @@
 		. = TRUE
 	take_damage(I.force, I.damtype, "melee")
 
-
-/obj/item/storage/attackby(obj/item/I, mob/user, params)
-	. = ..()
-	user.changeNext_move(CLICK_CD_FASTEST)
```

Comment:
Should check how this behavior is now handled, as there's a very notable delay when clicking on inventory slots now.